examples/calculate_spectrum_using_vaex.py

```diff
@@ -6,7 +6,7 @@
 Calculate spectrum using Vaex
 ===============================
 
-Calculating spectrum using pandas is memory intensive , that is it loads data into memory for calculating spectrum .For larger number of lines it restricts the 
+Calculating spectrum using pandas is memory intensive , that is it loads data into memory for calculating spectrum .For larger number of lines it restricts the
 number of lines that can be calculated on a device.
 When number of liens is more Vaex is recommended as engine.
 VAEX is a similar table management tool than PANDAS but uses memory mapping, a zero memory copy policy, and lazy computations for the best performance.
```

Comment:
```suggestion
Calculating a spectrum using pandas can be memory-intensive, as it loads data into memory for the computation. This can restrict the number of lines that can be processed on a device, especially for larger datasets. In such cases, Vaex is recommended as the engine. Vaex is a table management tool similar to PANDAS, but it utilizes memory mapping, a zero-memory-copy policy, and lazy computations for optimal performance.
```